### DISCUSSION TITLE
I want to translate GitHub into Korean.

### DISCUSSION THREAD

**Discussion by Astraevs-12** [community/community#14760] [Other Feature Feedback, Questions, & Ideas]
Hi.
First of all, thank you for creating this wonderful site.
I want to translate github into Korean.
I recently started using GitHub and it was difficult to learn because it was not translated.
So I want to contribute to the translation so that other people can use it easily.
I want you to allow this! 

Have a good day!

**Comment by Pudron** [comment]
Another solution may be creating a Korean mirror repository of this site.

**Reply by Astraevs-12** [reply]
I'm sorry to see you now.
Mirror sites can be a good alternative, but now I'm just a student.
But maybe I can try it in the future!
